Gemfile

```diff
@@ -2,8 +2,10 @@ source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.6'
-# Use MySQL as the database for Active Record
-gem 'mysql2'
+# Use Postgres as the database for Active Record
```

Comment:
Oh yeah you missed you announcement during class. Postgres doesn't play well with MySQL and strongly prefers Postgres. Postgres also has more powerful full text search anyway.